common/religion/religions/shamanism_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_crime
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_ignored
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_ignored
+	doctrine = doctrine_life_magic_accepted
```

Comment:
Life ignored.
There are no dwarven, orcish druids.
`earth_mother_worship`, `tidemother`, and `taunka_religion` may have it accepted.

Reply:
Done